installer/bundle/bundle_skel.sh

```diff
@@ -881,7 +887,8 @@ case "$installMode" in
             # Install SCX (and OMI)
             [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
             [ -n "${debugMode}" ] && DEBUG="--debug" || DEBUG=""
-            ./bundles/${SCX_INSTALLER} --install $FORCE $DEBUG
+            [ -n "${enableOMFlag}" ] && ENABLE_OM="--enable-opsmgr" || ENABLE_OM=""
+            ./bundles/${SCX_INSTALLER} --install $FORCE $DEBUG $ENABLE_OM
```

Comment:
Can you provide a link to the changes that are in the SCX installer bundle to support this flag?